basis/common-router/src/main/java/com/softgroup/common/router/api/AbstractRequestHandler.java

```diff
@@ -6,7 +6,7 @@
 import com.softgroup.common.protocol.Response;
 import com.softgroup.common.protocol.ResponseData;
 
-public class AbstractRequestHandler<T extends RequestData, R extends ResponseData> implements RequestHandler {
+public abstract class AbstractRequestHandler<T extends RequestData, R extends ResponseData> implements RequestHandler {
 	@Override
 	public String getName() {
 		return null;
```

Comment:
No. I will declare this method as abstract and will implemet it in inherited classes.